crits/emails/handlers.py

```diff
@@ -548,10 +545,6 @@ def handle_email_fields(data, analyst, method, related_id=None, related_type=Non
                               date=datetime.datetime.now())
         new_email.add_campaign(ec)
 
-
-    new_email.save(username=analyst)
```

Comment:
With this removed, when does it get saved to the database to get an ObjectId assigned so it can be used when forming any relationships necessary?